server.js

```diff
@@ -1,6 +1,14 @@
 require("dotenv").config();
 /* ==== External Modules ==== */
 const express = require("express");
+
+const bodyParser = require("body-parser");
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7821591)
```suggestion
const logger = require("morgan");
```